.github/workflows/npm_publish.yml

```diff
@@ -0,0 +1,47 @@
+name: NPM publish on new release
+
+on: 
+  release:
+    types: [created]
+    branches: 
+      - master
+
+jobs:
+  build:
```

Comment:
Is there a reason for having a job for the build step rather than putting `npm run build` after `npm ci` in the npm-publish step?

Reply:
hmm yeah you are right, it can just be under one job.